pkgs/by-name/wc/wcc/package.nix

```
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation {
   pname = "wcc-unstable";
-  version = "2018-04-05";
+  version = "2024-01-13";
```

Wrong year (should be 2025 now), and this doesn't follow our unstable versioning guidelines: https://github.com/NixOS/nixpkgs/blob/d37c24666dc05cdf1e9b76ea4437e3f222c7f944/pkgs/README.md#versioning

```suggestion
  pname = "wcc";
  version = "0.0.7-unstable-2025-01-13";
```